pkg/operation/botanist/operatingsystemconfig.go

```diff
@@ -191,11 +168,127 @@ func (b *Botanist) computeOperatingSystemConfigsForWorker(machineTypes []gardenv
 		"secretName":           secretName,
 		"customization":        customization,
 	}
+	var (
```

Comment:
What about the old, still existing `.spec.kubernetes.kubelet` section? I think we still have to consider this. If somebody defines the `kubelet` section in the worker pools then this will be taken, but if the `kubelet` section in the worker pool is `nil` then the `.spec.kubernetes.kubelet` config should be used. However, let's not add new fields to it. Just handle the still existing fields.

Reply:
Why not use the same kubelet struct for the worker pools and the .spec.kubernetes.kubelet? Then as you said, we use that as the default in case there is no config in the worker pool.

Reply:
Using the .spec.kubernetes.kubelet as default currently in case there is no kubelet section defined in the worker pool.